### ISSUE TITLE
Updated version?

### ISSUE THREAD

**Issue by terehov** [jrmuizel/qpdf.js#1]
I just stumbled upon your library. Can you maybe provide how you did it? I would like to compile a newer version. Thank you!

**Comment by andygock** [comment]
@terehov  I believe it was done using [Emscripten](https://emscripten.org/).

@jrmuizel Would you be able to share your resources of how you compiled so others can create an updated library?

**Comment by jrmuizel** [comment]
Yes, it was done with emscripten. I can't find any records from when I did it and don't remember anything. Sorry.
